app/src/main/res/layout/story_chapter_view.xml

```diff
@@ -41,10 +41,7 @@
           android:id="@+id/chapter_thumbnail"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          android:contentDescription="@{viewModel.name}"
-          android:paddingStart="40dp"
-          android:paddingEnd="40dp"
-          android:scaleType="centerCrop"
+          android:scaleType="centerInside"
           android:src="@{viewModel.chapterThumbnail.thumbnailGraphic}"
           app:layout_constraintDimensionRatio="H,4:3"
```

Comment:
I am not sure about this ratio here, let me confirm on this.

Reply:
The ratio here should be `16:9`

Reply:
I am also not sure about that. I have sent u updated screens on Hangout. PTAL at 3rd screen, the size of image is confusing me compared to first 2 screens. I think we have to put some restriction on height.

Reply:
@Luffy18346 Please add those screenshots here for everyone's reference. Also the ratio is 16:9.

Reply:
@rt4914 , I have updated the ratio to 16:9 and also added final screenshots or the work at the bottom